tests/test_citations.py

```diff
@@ -320,8 +421,8 @@ def test_cite_command_render_stdout(args, expected):
     """
     args = [
         'manubot', 'cite', '--render',
-        '--csl', 'https://github.com/greenelab/manubot-rootstock/raw/22b526073e0fe7a96cc35b612c47c371c92333df/build/assets/style.csl',
-        'arxiv:1806.05726v1', 'doi:10.6084/m9.figshare.5346577.v1',
+        '--csl', 'https://github.com/greenelab/manubot-rootstock/raw/3df864ecda8e22e738bf6f7f24833799b99cd961/build/assets/style.csl',
```

Comment:
Does this need to be updated to version `935539bc33510cabe9aeb36a4990b17e426ace4b` that you used in the example in https://github.com/greenelab/manubot/pull/58#issuecomment-427187132?  The examples in the comments look good, but I see that the test cases here still have the extra blank lines.

Reply:
Yes, but I was planning that we'd merge https://github.com/greenelab/manubot/pull/58 first, since if we squash that will change the commit hash.

Reply:
Makes sense.  I approved https://github.com/greenelab/manubot-rootstock/pull/134 so please proceed.  I'll approve this as well.